### PR TITLE
Documentation for metrics

### PR DESCRIPTION
**Pull request by KristopherJordan**
Some documentation for the beloved metrics (finally).

### CODE REVIEW

File: source/includes/_stats.md [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+    "sum_values": 1000.0, 
+    "_id": "d721daa2-46be-4685-8fea-b6a17b05fe88:foo.bar:1523297656", 
+    "time_stamp": {"$date": 1523297656}, 
+    "name": "foo.bar"
+}
+```
+
+Argument | Type | Description
+-------- | ---- | -------
+**name** | `string` | The metric key on which your metrics are grouped on. _Required field_
```
Comment (GabrielBlondeau-Kirby) [inline source/includes/_stats.md]: the ** already means is required, I think you don't need `_Required field_`

File: source/includes/_stats.md [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+    "from_date": "2018-04-07T00:00:00+00:00", 
+    "to_date": "2018-04-08T23:59:59.99999+00:00"}, 
+}
+
+```
+
+## Get Metric Count
+
+This endpoint returns the count of instances of a specific metric key between two dates.
+
```
Comment (GabrielBlondeau-Kirby) [inline source/includes/_stats.md]: just remove 1 or 2 lines here

File: source/includes/_stats.md [hunk 3]
```diff
@@ -0,0 +1,338 @@
+# Statistics
```
Comment (pedropregueiro) [inline source/includes/_stats.md]: Metrics

**Comment by pedropregueiro** [comment]
also check https://github.com/shareactorIO/backend/issues/1669

**Pull request merged**